excel_import_export/models/xlsx_export.py

```diff
@@ -0,0 +1,289 @@
+                if is_extend and not rows_inserted:
+                    rows_inserted = True
+                    st.insert_rows(row + 1, row_count - 1)
+                # --
+                for (row_val, style) in vals[field]:
+                    new_row = row + i
+                    new_rc = "{}{}".format(col, new_row)
+                    row_val = co.adjust_cell_formula(row_val, i)
+                    if row_val not in ("None", None):
+                        st[new_rc] = co.str_to_number(row_val)
+                    if style:
+                        styles = self.env["xlsx.styles"].get_openpyxl_styles()
+                        co.fill_cell_style(st[new_rc], style, styles)
+                    i += 1
+                # Add footer line if at least one field have sum
+                f = func.get(field, False)
+                if f and new_row > 0:
+                    new_row += 1
+                    f_rc = "{}{}".format(col, new_row)
+                    st[f_rc] = "={}({}:{})".format(f, rc, new_rc)
+                    styles = self.env["xlsx.styles"].get_openpyxl_styles()
+                    co.fill_cell_style(st[f_rc], style, styles)
+                cont_row = cont_row < new_row and new_row or cont_row
+        return
+
+    @api.model
+    def export_xlsx(self, template, res_model, res_ids):
+        if template.res_model != res_model:
+            raise ValidationError(_("Template's model mismatch"))
+        data_dict = co.literal_eval(template.instruction.strip())
+        export_dict = data_dict.get("__EXPORT__", False)
+        out_name = template.name
+        if not export_dict:  # If there is not __EXPORT__ formula, just export
+            out_name = template.fname
+            out_file = template.datas
+            return (out_file, out_name)
+        # Prepare temp file (from now, only xlsx file works for openpyxl)
+        decoded_data = base64.decodestring(template.datas)
+        ConfParam = self.env["ir.config_parameter"].sudo()
+        ptemp = ConfParam.get_param("path_temp_file") or "/tmp"
+        stamp = dt.utcnow().strftime("%H%M%S%f")[:-3]
+        ftemp = "{}/temp{}.xlsx".format(ptemp, stamp)
+        f = open(ftemp, "wb")
+        f.write(decoded_data)
+        f.seek(0)
+        f.close()
+        # Workbook created, temp file removed
+        wb = load_workbook(ftemp)
+        os.remove(ftemp)
+        # Start working with workbook
+        records = res_model and self.env[res_model].browse(res_ids) or False
+        outputs = []
+        for record in records:
+            self._fill_workbook_data(wb, record, export_dict)
+            # Return file as .xlsx
+            content = BytesIO()
+            wb.save(content)
+            content.seek(0)  # Set index to 0, and start reading
+            out_file = content.read()
+            if record and "name" in record and record.name:
+                out_name = record.name.replace(" ", "").replace("/", "")
```

Comment:
```suggestion
                out_name = record.name.replace(" ", "").replace("/", "")
```

Please, Could you add function get name this line for manual name export file excel?